apps/omg_watcher/test/omg_watcher/db/txoutput_test.exs

```diff
@@ -13,12 +13,16 @@
 # limitations under the License.
 
 defmodule OMG.Watcher.DB.TxOutputTest do
+  import Mox
```

Comment:
can you be more explicit in what you're importing?

Reply:
@InoMurko yes I can do that. Shall we continue with Mox or not?